module-options.nix

```diff
@@ -227,10 +227,10 @@ in
               # `treefmt --fail-on-change` is broken for purs-tidy; So we must rely
               # on git to detect changes. An unintended advantage of this approach
               # is that when the check fails, it will print a helpful diff at the end.
-              PRJ=$TMP/project
-              cp -r ${self} $PRJ
-              chmod -R a+w $PRJ
-              cd $PRJ
+              export PROJECT_ROOT=$TMP/project
```

Comment:
```suggestion
              export PRJ_ROOT=$TMP/project
```

Reply:
Actually relative paths work in this project as well. I don't think we need this environment variable.